src/components/rangeUsePlanPage/versionsList/VersionsDropdownList.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import Box from '@material-ui/core/Box'
+import Collapse from '@material-ui/core/Collapse'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import moment from 'moment'
+import classnames from 'classnames'
+import Status from '../../common/Status'
+
+const VersionsDropDownList = ({ versions, open }) => {
+  const versionOptions = versions.map(v => ({
+    key: v.version,
+    value: v,
+    text: `v${v.version}`,
+    content: (
+      <div
+        style={{
+          display: 'grid',
+          gridTemplateColumns: '40px repeat(2, auto)',
+          gridTemplateRows: '1fr'
```

Comment:
Can we move all this inline styling to either one of the `scss` files, or just use material-ui's CSS-in-JS API?

Reply:
Yes, implemented in these commits: 501e0bd b55c187